sealtk/core/KwiverPipelinePortSet.cpp

```diff
@@ -0,0 +1,69 @@
+/* This file is part of SEAL-TK, and is distributed under the OSI-approved BSD
+ * 3-Clause License. See top-level LICENSE file or
+ * https://github.com/Kitware/seal-tk/blob/master/LICENSE for details. */
+
+#include <sealtk/core/KwiverPipelinePortSet.hpp>
+
+namespace sealtk
+{
+
+namespace core
+{
+
+namespace // anonymous
+{
+
+// ----------------------------------------------------------------------------
+void bind(std::string& out, std::string const& expected, std::string const& in)
+{
+  if (in == expected)
+  {
+    out = expected;
+  }
+}
+
+}
+
+// ----------------------------------------------------------------------------
+void KwiverPipelinePortSet::bind(
+  kwiver::embedded_pipeline& pipeline, int index, PortType type,
+  std::vector<PortReference> additionalPorts)
+{
+  auto const timePortName = this->portName("timestamp", index);
```

Comment:
Can we expect that all pipelines will have a `timestamp` port?

Reply:
That is an assumption we are making, yes.